src/base/utils/gzip.h

```diff
@@ -36,8 +36,8 @@ namespace Utils
 {
     namespace Gzip
     {
-        bool compress(QByteArray src, QByteArray &dest);
-        bool uncompress(QByteArray src, QByteArray &dest);
+        QByteArray compress(const QByteArray &data, bool *ok = nullptr, int level = 6);
```

Comment:
>Why it is optional then? 

to be consistent with `decompress()` ?
was suggested in: https://github.com/qbittorrent/qBittorrent/pull/6654#discussion_r112192031